src/backend/dx11/src/lib.rs

```diff
@@ -3412,15 +3409,82 @@ impl Image {
     }
 }
 
-#[derive(Clone)]
 pub struct ImageView {
     subresource: UINT,
     format: format::Format,
-    rtv_handle: Option<ComPtr<d3d11::ID3D11RenderTargetView>>,
-    srv_handle: Option<ComPtr<d3d11::ID3D11ShaderResourceView>>,
-    dsv_handle: Option<ComPtr<d3d11::ID3D11DepthStencilView>>,
-    rodsv_handle: Option<ComPtr<d3d11::ID3D11DepthStencilView>>,
-    uav_handle: Option<ComPtr<d3d11::ID3D11UnorderedAccessView>>,
+    rtv_handle: Option<*mut d3d11::ID3D11RenderTargetView>,
+    srv_handle: Option<*mut d3d11::ID3D11ShaderResourceView>,
+    dsv_handle: Option<*mut d3d11::ID3D11DepthStencilView>,
+    rodsv_handle: Option<*mut d3d11::ID3D11DepthStencilView>,
+    uav_handle: Option<*mut d3d11::ID3D11UnorderedAccessView>,
+    borrowed: bool,
```

Comment:
nit: perhaps turning this around and making this an `owned` flag would make sense, since it has additional semantics of acquiring/releasing ownership